daffodil-core/src/main/scala/org/apache/daffodil/dpath/Expression.scala

```diff
@@ -1642,10 +1642,10 @@ case class FunctionCallExpression(functionQNameString: String, expressions: List
           DFDLXBitAnd(_,_))
       case (RefQName(_, "bitOr", DFDLX), args) =>
         DFDLXBitExpr(functionQNameString, functionQName, args,
-          DFDLXOr(_,_))
+          DFDLXBitOr(_,_))
       case (RefQName(_, "bitNot", DFDLX), args) =>
-        DFDLXBitNot(functionQNameString, functionQName, args,
-          DFDLXNot(_,_))
+        DFDLXBitExprNot(functionQNameString, functionQName, args,
```

Comment:
Minor thought looking at this one last time. It might make sense to rename `DFDLXBitExpr` to `DFDLXBitBinaryExpr` and `DFDLXBitExprNot` to `DFDLXBitUnaryExpr`. Those two renames make the difference between the two class more clear (i.e. binary vs unary). And it also makes it clear that Unary isn't necessarily specific to NOT, even though that's currently the only thing we use it for. Maybe someday we'll add NAND, NOR or some other binray or unary operations, and it will be more clear which of DFDLXBitBinaryExpr or DFDLXBitUnary is the right thing to use.